script/Params.s.sol

```diff
@@ -50,7 +48,7 @@ uint256 constant PROPORTIONAL_GAIN = 222_002_205_862;
 uint256 constant INTEGRAL_GAIN = 16_442;
 
 // Job Params
-uint256 constant JOB_REWARD = 5e18; // 5 HAI
+uint256 constant JOB_REWARD = 5e18; // 5 OD
```

Comment:
Please set this to 0.20 OD

Reply:
`
// Job Params
uint256 constant JOB_REWARD = 0.2e18; // 0.2 OD
`